### DISCUSSION TITLE
Welcome to dotfiles Discussions!

### DISCUSSION THREAD

**Discussion by JDeepD** [Nova-Striker/dotfiles#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mastermind-sap** [comment]
Github got so irritated by our using github issues as whatsapp chat that it created discussions :rofl: !! @JDeepD
